src/mixins/kpiProgress.js

```diff
@@ -0,0 +1,74 @@
+import { mapState } from 'vuex';
+import { db } from '@/config/firebaseConfig';
+import { formatKPIValue, getKPIProgress, getKPIProgressQuery } from '@/util/kpiHelpers';
+
+export default {
+  data: () => ({
+    progressCollection: [],
+    goals: [],
```

Comment:
Brrr, jeg er ikke så glad i mixins, men hvis vi skal ha den, kunne vi prøvd å minimere/rendyrke den ved å holde for eksempel `goals` her og `setGoals()` nedenfor utenfor? De virker ganske uavhengige av resten av "progress"-tingene her?